src/Illuminate/Database/Console/Migrations/MigrateMakeCommand.php

```diff
@@ -70,7 +70,7 @@ public function handle()
         // It's possible for the developer to specify the tables to modify in this
         // schema operation. The developer may also specify if this table needs
         // to be freshly created so we can create the appropriate migrations.
-        $name = Str::snake(trim($this->input->getArgument('name')));
+        $name = Str::snake(trim(str_replace('-', ' ', $this->input->getArgument('name'))));
```

Comment:
For readability maybe it should be considered to chain the functions like
`Str::of($this->input->getArgument('name'))->replace('-', ' ')->trim()->snake();`.